examples/petclinic/types.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/http;
+
+public type ServerConfig record {
+    string host;
+    int port;
+};
+
+public type OwnerCreated record {|
+    *http:Created;
```

Comment:
We can configure the body as well right?

```ballerina
public type CreatedBody record {|
    int insertedId;
|};

public type OwnerCreated record {|
    *http:Created;
    CreatedBody body;
|};
```

Reply:
Having `anydata` is fine but that is more generic right? Here what we sent out is only the `insertId`. When we define a type for that, then we are specifically mentioned that we are returning a `insertId`. This makes sure where ever we use this type we are correctly adding the required fields (type based validation at compile time) and any consumers of this API knows we are sending specific fields so that they do not need to check whether the payload has a specific structure or not (API designing should be self-descriptive and Ballerina's type system makes sure of it)